ff/serving/python_client/sdk/register.py

```diff
@@ -0,0 +1,115 @@
+
+files_to_register = []
```

Comment:
Since these are so similar imo they should be in a dictionary of lists (unregisteredResources{}), which the key being which resource type (entity, feature, etc).